src/main/java/column/store/csv/CSVFile.java

```diff
@@ -0,0 +1,27 @@
+package column.store.csv;
```

Comment:
true, previously they were in the same package, but I decided to move them to own packages so that our implementations have equivalent/similar package structures